### ISSUE TITLE
Support for inline links

### ISSUE THREAD

**Issue by adamstep** [Instawork/hyperview#307]
See screenshot attached - implementation of such UI isn't quite possible at the moment.

A typical React Native implementation for this UI would be:

```
<Text>
  Includes deduction to cover Occupational Accident Insurance.
  <TouchableOpacity onPress={onPressMoreInfo}>
    <Text style={link}>More Info here</Text>
  </TouchableOpacity>
</Text>
```

which we could try to translate in HXML to

```
<text>
  Includes deduction to cover Occupational Accident Insurance.
  <text style="link" href="/more-info" action="push" trigger="press">More Info here</text>
</text>
```

However, this triggers a React Native error, as the added "href" internally wraps the inner link text component inside a view, and React Native requires <View> nested within <Text> to have a width and height defined.

Since the text needs to flow based on its content, we cannot infer the width and height, so we should refactor the `addHref` helper in hyperview to support this use case.

This somewhat relates to #288

**Comment by terryatgithub** [comment]
we've encountered this issue now as below, looking forward to an early resolution.
![image](https://user-images.githubusercontent.com/12246394/130736694-33f8b0df-8d7f-4582-ae8f-a4b06f41ab6d.png)

**Comment by flochtililoch** [comment]
Fixed by #462
